website/static/downloads/swagger.yaml

```diff
@@ -1366,7 +1374,70 @@ components:
       in: query
       description: Filter by target public key.
       schema:
-        $ref: '#/components/schemas/publicKeys'    
+        $ref: '#/components/schemas/publicKeys'   
+
+    addressQuery:
+      name: address
+      in: query
+      description: Address of the account.
+      schema:
+        type: string
+        format: hex
+    embeddedQuery:
+      name: embedded
+      in: query
+      description: If transaction if embedded.
```

Comment:
`If transaction is embedded.`?

Reply:
May I know if embedded means that it is an aggregate transaction?

Reply:
not really. That is a transaction that is in an aggregate transaction

Reply:
for example, route `/transaction/confirmed` returns next (hard simplification):
```
"aggregateTrnasaction" : {}
```
`/transaction/confirmed?embedded=true` returns next
```
"aggregateTrnasaction" : {}
"innerTransactionInAggregateTrnasaction" : {}
"innerTransactionInAggregateTrnasaction" : {}
```

Reply:
okay understood, thank you!

Reply:
okay understood, thank you for the explanation!